deploy/olm-catalog/kubefed-operator/4.2/kubefed-operator-ClusterPropagatedVersion_crd.yaml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Does it make sense to keep the file names consistent with the changes @font has made in https://github.com/openshift/kubefed-operator/pull/50

Reply:
Fixed.